waspc/src/Wasp/Generator/ServerGenerator/OperationsRoutesG.hs

```diff
@@ -72,40 +75,45 @@ operationsRoutesDirInServerSrcDir = [reldir|routes/operations/|]
 operationsRoutesDirInServerRootDir :: Path' (Rel C.ServerRootDir) (Dir OperationsRoutesDir)
 operationsRoutesDirInServerRootDir = C.serverSrcDirInServerRootDir </> operationsRoutesDirInServerSrcDir
 
-operationRouteFileInOperationsRoutesDir :: Wasp.Operation.Operation -> Path' (Rel OperationsRoutesDir) File'
-operationRouteFileInOperationsRoutesDir operation = fromJust $ SP.parseRelFile $ Wasp.Operation.getName operation ++ ".js"
+operationRouteFileInOperationsRoutesDir :: AS.Operation.Operation -> Path' (Rel OperationsRoutesDir) File'
+operationRouteFileInOperationsRoutesDir operation = fromJust $ SP.parseRelFile $ AS.Operation.getName operation ++ ".js"
 
 relPosixPathFromOperationsRoutesDirToSrcDir :: Path Posix (Rel OperationsRoutesDir) (Dir C.ServerSrcDir)
 relPosixPathFromOperationsRoutesDirToSrcDir = [reldirP|../..|]
 
-genOperationsRouter :: Wasp -> FileDraft
-genOperationsRouter wasp
+genOperationsRouter :: AppSpec -> FileDraft
+genOperationsRouter spec
   -- TODO: Right now we are throwing error here, but we should instead perform this check in parsing/analyzer phase, as a semantic check, since we have all the info we need then already.
   | any isAuthSpecifiedForOperation operations && not isAuthEnabledGlobally = error "`auth` cannot be specified for specific operations if it is not enabled for the whole app!"
-  | otherwise = C.makeTemplateFD tmplFile dstFile (Just tmplData)
+  | otherwise = C.mkTmplFdWithDstAndData tmplFile dstFile (Just tmplData)
   where
     tmplFile = C.asTmplFile [relfile|src/routes/operations/index.js|]
     dstFile = operationsRoutesDirInServerRootDir </> [relfile|index.js|]
     operations =
-      map Wasp.Operation.ActionOp (Wasp.getActions wasp)
-        ++ map Wasp.Operation.QueryOp (Wasp.getQueries wasp)
+      map (uncurry AS.Operation.ActionOp) (AS.getActions spec)
+        ++ map (uncurry AS.Operation.QueryOp) (AS.getQueries spec)
```

Comment:
Ah a use of `uncurry` in the wild, love it! :D

Reply:
Ok I have to admit, this was suggested by the LSP code action, but I completely agreed with it :D.